src/main/java/at/hannibal2/skyhanni/config/Storage.java

```diff
@@ -468,6 +468,8 @@ public static class DianaStorage {
             // TODO renmae
             public MythologicalCreatureTracker.Data mythologicalMobTracker = new MythologicalCreatureTracker.Data();
 
+            @Expose
+            public List<LorenzVec> foundBurrowLocations = new ArrayList<>();
```

Comment:
Should this be in profile specific or just in general, in general makes more sense to me, also if it is expected to get very long I feel it should be separated to a new config file

Reply:
Done.
When the list gets too long, we can write a smart "extract on file io actions" logic without damaging the usage of the config structure. (Something we should have used for known features and Jacob contests, and friends as well)